### PR TITLE
drop account from update

### PR DESCRIPTION
**Pull request by mgold1234**
Signed-off-by: mgold1234 <mgold@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/routes/updates.go [hunk 1]
```diff
@@ -266,7 +263,7 @@ func AddUpdate(w http.ResponseWriter, r *http.Request) {
 	respondWithJSONBody(w, ctxServices.Log, updates)
 }
 
-// GetUpdateByID obtains an update from the database for an account
+// GetUpdateByID obtains an update from the database for an ordID
```
Comment (de1987) [inline pkg/routes/updates.go:266]: typo

**Comment by de1987** [comment]
For the `DispatcherPayload`, I think we need to the drop `Account` field or change it for `json:"account,omitempty"`.
```
type DispatcherPayload struct {
	Recipient   string `json:"recipient"`
	PlaybookURL string `json:"url"`
	Account     string `json:"account"`
	OrgID       string `json:"org_id"`
}
```

**Why do we need to?**
Currently, our device updating is failing and the error it's getting is:
`"error":"error calling playbook dispatcher, got status code 400 and body {\"message\":\"Request body has an error: doesn't match the schema: Error at \\\"/0/account\\\": Minimum string length is 1\"}\n"`

Does it make sense?

**Comment by de1987** [comment]
@mgold1234 could you please, check that above?

**Comment by mgold1234** [comment]
work has been transferred to another PR - https://github.com/RedHatInsights/edge-api/pull/1311

**Pull request closed without merging**